trieste/acquisition/function/function.py

```diff
@@ -17,6 +17,7 @@
 """
 from __future__ import annotations
 
+import math
```

Comment:
no need for this here, right?

Reply:
we need math.pi for calculating the BALD, is there another alternative?

Reply:
I'm saying its now in a wrong file, right?

Reply:
sorry my bad